extensionTask.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+if [ "$1" == "begin" ]; then
```

Comment:
This script will not be accessible in the build definitions until the cloning has happened so we cannot depend on that. Also I don't like checking this into every repo (especially in the root of the repo).

Reply:
I suggest using an inline bash script for the begin/end steps.

Reply:
I just did not find a CMD task where one could write the script inline, hence my recommendation. If there is one where this can be done, we should use it by all means

Reply:
Oh this is cool!Thanks for the pointer